velox/functions/sparksql/Arithmetic.h

```diff
@@ -219,6 +219,18 @@ struct HypotFunction {
   }
 };
 
+template <typename T>
+struct Log2Function {
+  FOLLY_ALWAYS_INLINE bool call(double& result, double a) {
+    double yAsymptote = 0.0;
+    if (a <= yAsymptote) {
```

Comment:
Would you document this behavior?

Reply:
Does Spark return null for zero input? Would you double check?

Reply:
Thanks, Just double check it.
scala> spark.sql(" SELECT log2(0);").show()
+-------+
|LOG2(0)|
+-------+
|   null|
+-------+

scala> spark.sql(" SELECT log10(0);").show()
+--------+
|LOG10(0)|
+--------+
|    null|
+--------+

Reply:
And document are updated too.